pkg/controller/user/create.go

```diff
@@ -111,6 +111,9 @@ func (c *Controller) HandleCreate() http.Handler {
 			return
 		}
 
+		if existing != nil && existing.ID != 0 {
+			flash.Alert("A user with this email address already exists, so the exiting name has been used.")
```

Comment:
I intentionally didn't do this because it leaks information. This tells the admin with UserWrite permissions that the person they just added was already a member of another realm.

Reply:
fair point - removed.